src/v2/guide/migration.md

```diff
@@ -379,48 +381,54 @@ methods: {
 
-<p class="tip">Note that for other directives (e.g. `v-if` and `v-show`), JavaScript's normal truthiness still applies.</p>
+<p class="tip">
+  Tenga en cuenta que para otras directivas (por ejemplo, `v-if` y `v-show`), la veracidad normal de JavaScript aún se aplica.
+</p>
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run your end-to-end test suite, if you have one. The <strong>failed tests</strong> should alert to you to any parts of your app that may be affected by this change.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute sus pruebas de extremo a extremo, si tiene una. Las <strong>pruebas fallidas</strong> le alertarán sobre cualquier parte de su aplicación que pueda ser afectada por este cambio.
+  </p>
 </div>
 {% endraw %}
 
-### Listening for Native Events on Components with `v-on` <sup>changed</sup>
+### Escuchar eventos nativos en componentes con `v-on` <sup>cambiado</sup>
 
-When used on a component, `v-on` now only listens to custom events `$emit`ted by that component. To listen for a native DOM event on the root element, you can use the `.native` modifier. For example:
+Cuando se usa en un componente, `v-on` ahora solo escucha eventos personalizados (`$emit`) emitidos por ese componente. Para escuchar un evento del DOM nativo en el elemento raíz, usted puede usar el modificador `.native`. Por ejemplo:
 
 ``` html
 <my-component v-on:click.native="doSomething"></my-component>
 ```
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run your end-to-end test suite, if you have one. The <strong>failed tests</strong> should alert to you to any parts of your app that may be affected by this change.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute sus pruebas de extremo a extremo, si tiene una. Las <strong>pruebas fallidas</strong> le alertarán sobre cualquier parte de su aplicación que pueda ser afectada por este cambio.
+  </p>
 </div>
 {% endraw %}
 
-### `debounce` Param Attribute for `v-model` <sup>removed</sup>
+### `debounce` Atributo de parámetro para `v-model` <sup>eliminado</sup>
 
-Debouncing is used to limit how often we execute Ajax requests and other expensive operations. Vue's `debounce` attribute parameter for `v-model` made this easy for very simple cases, but it actually debounced __state updates__ rather than the expensive operations themselves. It's a subtle difference, but it comes with limitations as an application grows.
+_Debouncing_ se usa para limitar la frecuencia con la que ejecutamos _requests_ Ajax y otras operaciones costosas. El parámetro de atributo `debounce` de Vue para `v-model` lo hizo fácil para casos muy simples, pero en realidad no dio lugar a __actualizaciones de estado__ en lugar de las costosas operaciones en sí mismas. Es una diferencia sutil, pero viene con limitaciones a medida que aumenta la complejidad de la aplicación.
```

Comment:
What do you about translating `_requests`_ to `solicitudes`?
```suggestion
_Debouncing_ se usa para limitar la frecuencia con la que ejecutamos solicitudes Ajax y otras operaciones costosas. El parámetro de atributo `debounce` de Vue para `v-model` lo hizo fácil para casos muy simples, pero en realidad no dio lugar a __actualizaciones de estado__ en lugar de las costosas operaciones en sí mismas. Es una diferencia sutil, pero viene con limitaciones a medida que aumenta la complejidad de la aplicación.
```

Reply:
I was in doubts regarding that one.. haha

I'm good changing it, but I'd add it to the list of words-mapping so we keep consistent with this kind of translations!

Thank you